tex/chapters/07-ApplyingModel.tex

```diff
@@ -195,3 +204,16 @@ \section{Applying Model to Axelrod-Python}
 AntiCycler follows a sequence of plays that has no cycles: $CDD\ CD\ CCD\ CCCD\ CCCCD \dots $, whereas the QLearners use Q-Learning \cite{Watkins1992} in an attempt to find an optimal solution.
 It is understandable that different Q-Learners may be predicted equivalent by the model, as it is possible that find the same solution.
 However, without knowledge of the solution they find it is unclear why they are considered similar to AntiCycler and this would require further work.
+
+
+
+\section{Conclusion}
+
+Initially an estimation of the sensitivity of the model was obtained using memory one strategies and it was found the model achieved good results when the deviation was $\delta > 0.3$.
+Then the model was applied to Axelrod-Python to determine whether the library contained any duplicate strategies.
+Overall the model found 124 different neighbourhoods.
+98 neighbourhoods contained only 1 strategy, ie. the strategies were unique.
+15 neighbourhoods contained 2 strategies, 4 contained 3 strategies, 1 neighbourhood had 4 strategies, 2 neighbourhoods had 5 strategies, 2 had 6, 1 neighbourhood contained 7 strategies and the largest neighbourhood contained 8.
+
+Many of the neighbourhoods contained strategies that were logical for the model to identify as identical.
+The remaining neighbourhoods will require more analysis to determine whether the strategies they contain are actually equivalent or not.
```

Comment:
One approach could be to compare the ashlock fingerprint data for these (calculated with the same seed).